cmd/defenseclaw-setup/platform_other.go

```
@@ -7,28 +7,77 @@ package main
-}
-func removeUserPath(_ string, _ bool) error             { return errors.New("Windows-only operation") }
-func registerInstalledApp(_, _, _ string, _ bool) error { return errors.New("Windows-only operation") }
-func unregisterInstalledApp() error                     { return errors.New("Windows-only operation") }
+func acquireSetupLock() (func() error, error) {
+	return func() error { return nil }, nil
+}
+func rejectReparseAncestors(_ string) error { return nil }
+func rejectReparseExisting(_ string) error  { return nil }
+func isReparsePoint(_ string) (bool, error) { return false, nil }
+func addUserPath(_ string) (bool, bool, bool, error) {
+	return false, false, false, errors.New("Windows-only operation")
```

golangci-lint (2.12.2)

[error] ST1005: error strings should not be capitalized

(staticcheck)